cli/gencert/gencert.go

```diff
@@ -112,6 +112,13 @@ func gencertMain(args []string, c cli.Config) (err error) {
 			return err
 		}
 
+		if len(req.Hosts) == 0 {
```

Comment:
This would also benefit from the previously-mentioned comment. Such a comment will make it easier to update the version and section of the BR in future revisions.